lib/cretonne/src/verifier.rs

```diff
@@ -614,6 +615,28 @@ impl<'a> Verifier<'a> {
         Ok(())
     }
 
+    fn cfg_integrity(&self, cfg: &ControlFlowGraph) -> Result<()> {
+        for ebb in self.cfg.ebbs() {
```

Comment:
I think you should use `layout.ebbs()` instead here. The `cfg.ebbs()` method probably shouldn't be public.